src/Roots/Acorn/ComposerScripts.php

```diff
@@ -15,9 +13,18 @@ class ComposerScripts extends FoundationComposerScripts
      */
     protected static function clearCompiled()
     {
-        $console = new Console(new Filesystem(), getcwd());
+        $laravel = new \Roots\Acorn\Application(getcwd(), ['bootstrap' => getcwd() .'/storage/framework']);
```

Comment:
This path should not be hard-coded. Users can change the path by setting `ACORN_BOOTSTRAP_PATH` or calling `Application::useBootstrapPath()`. So we shouldn't assume any specific value or pattern.

You can try using the trait [`GetsFreshApplication::getFreshApplication()`](https://github.com/roots/acorn/blob/main/src/Roots/Acorn/Console/Concerns/GetsFreshApplication.php) if you want. Seems like that might be what you're going for here.

Reply:
ooh i didnt know